src/app/campaign-list/list/filter-date.component.scss

```diff
@@ -0,0 +1,10 @@
+/* have to override the global button styles */
```

Comment:
We should really remove global button styles and move them to scoped component styles if we're getting serious about using material components.

Reply:
I also saw that madness, in the datepicker.  Good monkey patch!